src/mail/mail.service.ts

```diff
@@ -29,11 +25,54 @@ export class MailService {
       const message: MailDataRequired = {
         from,
         to: email,
-        templateId: this.winnerTemplateId,
+        templateId: this.configService.get<string>(
+          'SENDGRID_TEMPLATE_LOOT_CLAIMED_ID',
+        ),
         dynamicTemplateData: {
           loot_img_url: imgUrl,
-          loot_display_name: displayName,
+          loot_display_name: name,
+          subject,
+        },
+      };
+      const [mailResponse] = await sgMail.send(message);
+      if (mailResponse.statusCode == 202) {
+        return true;
+      } else {
+        return false;
+      }
+    } catch (error) {
+      throw new MailError(MailError.SERVER_CODES.INTERNAL_SERVER_ERROR, error);
+    }
+  }
+
+  async sendRedeemedLootConfirmation(
+    email: string,
+    firstName: string,
+    lastName: string,
+    lootName: string,
+    deliveryAddress?: DeliveryAddress,
+  ): Promise<boolean> {
+    try {
+      const subject =
+        'Order confirmation success! You have ordered your exclusive gift 🎉';
```

Comment:
Replace with: "Congratulations! Your order #00006 is confirmed 🎉" 

The order number should be an auto incremented field on the Order table. It's mostly used for tracking and is easier to handle than uuid when interacting with users. Order number should be padded with leading zeros to have a lenght of 6.